prow/config.yaml

```diff
@@ -171,6 +171,11 @@ branch-protection:
             users: []
             teams:
             - stage-bots
+        kubectl:
```

Comment:
The `kubectl` key is already included on line 141.

Reply:
Removed this since travis won't be used anymore (kubectl is now a staging repo)